app/views/items/index.html.erb

```diff
@@ -0,0 +1,12 @@
+<h1>Items</h1>
+<% @items.each do |item| %>
+  <div id = "item-<%= item.id %>">
+    <h3>List of Added Items:</h3>
```

Comment:
I don't know if the intention is to have "List of Added Items:" printed once for each item but if it isn't then I think you can move it out of the loop so it prints just once :)